generators/base/types.d.ts

```diff
@@ -24,5 +24,5 @@ export type Control = BaseApplicationControlProperties & {
    * cleanupFiles({ '6.0.0': ['file1', 'file2', [application.shouldRemove, 'file3']] })
    */
   cleanupFiles: (cleanup: Record<string, (string | [boolean, ...string[]])[]>) => Promise<void>;
-  getWebappTranslation?: (s: string, data?: Record<string, any>) => string;
+  getWebappTranslation: (s: string, data?: Record<string, any>) => string;
```

Comment:
For this api to exist, languages generator should be composed with.
It's probably better to keep it optional.

Reply:
It's not: following the code, leaving it undefined will lead to exception

Reply:
It’s for blueprints, required for angular, react and vue since they composes with languages generator everytime.